docs/modules/operation/pages/deep-dive/kafka-producer/configure-kafka.adoc

```diff
@@ -169,3 +173,25 @@ admin@opennms()> config:update
 ----
 
 In the example above, we set the vertex and edge topics to be different by default.
+
+
+== Configure the alarm sync streams client
+
+When producing alarms to a topic, we also automatically enable a Kafka Stream client to help synchronize the contents of the topic and ensure that it remains eventually consistent with the database.
```

Comment:
```suggestion
When producing alarms to a topic, we also automatically enable a Kafka Stream client to help synchronize the contents of the topic and ensure that it eventually remains consistent with the database.
```
For my own understanding, does this mean it takes noticeable time to synchronize?

Reply:
Yeah, there's some lag, and it will get there, eventually. 

Given that "[eventual consistency](https://en.wikipedia.org/wiki/Eventual_consistency)" is a known term, should these stay together?

Reply:
Ah, should have checked. In that case I'd suggest:

```suggestion
When producing alarms to a topic, we also automatically enable a Kafka Stream client to help synchronize the contents of the topic and ensure eventual consistency with the database.
```